modules/persistent-storage-rhv.adoc

```diff
@@ -0,0 +1,72 @@
+[id="persistent-storage-rhv_{context}"]
+= Creating a persistent volume on RHV
+
+When you create a `PersistentVolumeClaim` (PVC) object, {product-title} provisions a new persistent volume (PV) and creates a `PersistentVolume` object.
+
+.Prerequisites
+* You are logged in to a running {product-title} cluster.
+* You provided the correct {rh-virtualization} credentials in  `ovirt-credentials` secret.
+* You have installed the oVirt CSI Driver.
+* You have defined at least one storage class.
+
+.Procedure (UI)
+To dynamically create a persistent volume on {rh-virtualization} using the web console:
+
+. In the {product-title} console, click *Storage* -> *Persistent Volume Claims*.
+. In the persistent volume claims overview, click *Create Persistent Volume Claim*.
+. Define the required options on the resulting page.
+. Select the appropriate `StorageClass` (`ovirt-csi-sc` by default).
+. Enter a unique name for the storage claim.
+. Select the access mode, currently RWO (ReadWriteOnce) is the only supported access mode.
+. Define the size of the storage claim.
+. Select the Volume Mode:
++
+`Filesystem` - mounted into pods as a directory (default)
++
+`Block` - block device, without any file system on it
++
+. Click *Create* to create the `PersistentVolumeClaim` and generate a `PersistentVolume`.
+
+.Procedure (CLI)
+
+To dynamically create a {rh-virtualization} CSI volume using the command-line interface (CLI):
+
+. Create and save a file with the `PersistentVolumeClaim` object described by the following YAML:
++
+.pvc-ovirt.yaml
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc-ovirt
```

Comment:
Should it always have this `name`?

Reply:
no this is an example

Reply:
If this is an example, it needs to be clearer, like: 
```
  name: <pvc_name>
```

I suggest pairing it with an annotation.

Reply:
added the word sample in 2 modules for the sentence "...described by the following YAML". I see almost no such usage in the rest of the OCP docs, but users generally understand that almost all Ansible playbooks that are presented are samples/examples/templates that need to be modified, otherwise, we would ask them to simply call up and run the playbook.